visualization/app/codeCharta/util/nodePathHelper.ts

```diff
@@ -26,5 +26,5 @@ export function getParent<T>(hashMap: Map<string, T>, path: string): T {
 		if (node) {
 			return node
 		}
-	} while (path !== CodeChartaService.ROOT_PATH && path.length)
+	} while (path !== CodeChartaService.ROOT_PATH && path.length > 0)
```

Comment:
how did this work in the past, seems like we never checked correctly?

Reply:
It is identical, it just looks nicer.

Reply:
Ah ok, I didn't know that thx